### ISSUE TITLE
k8s_core_v1_secret resource leaks sensitive data in string_data attribute

### ISSUE THREAD

**Issue by UncleIS** [mingfang/terraform-provider-k8s#14]
Hi! 

First of all, thanks for the great provider! I've been using it for quite some time, and it's a great piece of software! 

The issue that I'm having seems to be the continuation of #12.

We are using Terraform 0.14 and k8s provider version 1.0.1. When we plan Terraform execution, the plan command prints all the data in `data.sting_data` field to STDOUT:

```
 # k8s_core_v1_secret.my_secret will be created
  + resource "k8s_core_v1_secret" "my_secret" {
      + data        = (sensitive value)
                            #^^^^^^^^^^^^^^^
                            # This part is really secret
      + id          = (known after apply)
      + immutable   = (known after apply)
      + string_data = {
          + "text" = "my_secret_string"
                           #^^^^^^^^^^^^^^^^^^^
                           # But this part prints secret to STDOUT
        }
      + type        = "Opaque"
      ....
}
```

I'm not sure about that, but I guess that printing sensitive data in clear text is not the intended behavior, is it?   

Thanks!

**Comment by mingfang** [comment]
@UncleIS 
Thanks for your kind words, and your support.
This issue is very important and I will get it fixed over the weekend.

**Comment by mingfang** [comment]
fixed in v1.02
https://github.com/mingfang/terraform-provider-k8s/releases/tag/v1.0.2
